infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -93,6 +95,12 @@
      */
     private Future failureDetectorFuture = null;
     private Future healingDetectorFuture = null;
+
+    /**
+     * Recovery attempts.
+     */
+    private static final int RECOVERY_RETRIES = 3;
```

Comment:
Yeah, make this part of the options map and default it to 3

Reply:
Fixed